testing/endtoend/backend/anvil_local_test.go

```diff
@@ -45,11 +45,11 @@ func TestLocalAnvilStarts(t *testing.T) {
 	require.NoError(t, err)
 
 	// There should be at least 100 blocks
-	bn, err2 := a.Client().HeaderByNumber(ctx, nil)
+	bn, err2 := a.Client().HeaderNumberUint64(ctx, nil)
 	if err2 != nil {
 		t.Fatal(err2)
 	}
-	if bn.Number.Uint64() < 100 {
+	if bn < 100 {
```

Comment:
Is this because the min assertion period blocks is 100? I think we might not need to do this if we tweak some configs. It could to faster test times

Reply:
I don't know.